.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+** Version and configuration  information (please complete the following information):**
+ - Host computer OS and Version: [e.g. Windows 10, macOS High Sierra]
+ - Docker version information (use `docker version`) and insert it here.
+ - ddev version information (use `ddev version`)
+ - config.yaml contents for the misbehaving project
+ - Do you have any custom configuration (nginx, php, mysql) in the .ddev folder?

```

Comment:
At some point, we may want to direct people to https://supportdetails.com/ for system info. Or have a more verbose version for `ddev version --bug-report` to find and spit all this info out.

Reply:
I sure like that idea! `ddev version --bug-report`!